docs/faq.md

```diff
@@ -0,0 +1,17 @@
+# FAQ
+
+**Is Shift Available on all Operating Systems?**
+
+Shift is available for all major 64-bit operating systems including Windows and Linux. MacOS compatability testing is currently in progress.
```

Comment:
Is there a minimum Windows version required?
For linux, on what distribution has this been tested? Ubuntu, Debian, Centos, ...

Reply:
@mromeoinbibo @craigskerryinbibo ?